examples/models/models_first_person_maze.nim

```diff
@@ -0,0 +1,87 @@
+#[*******************************************************************************************
+*
+*   raylib [models] example - first person maze
+*
+*   This example has been created using raylib 2.5 (www.raylib.com)
+*   raylib is licensed under an unmodified zlib/libpng license (View raylib.h for details)
+*
+*   Copyright (c) 2019 Ramon Santamaria (@raysan5)
+*   Adapted to Nim by jseb (2021)
+*
+********************************************************************************************]#
+
+import nimraylib_now
+
+proc main() =
+  const
+    screenWidth = 1024
+    screenHeight = 768
+
+  initWindow(screenWidth, screenHeight, "raylib [models] example - first person maze")
+
+  var camera = Camera3D(
+    position: (0.2, 0.4, 0.2), target: (0.0, 0.0, 0.0), up: (0.0, 1.0, 0.0), fovy: 45.0, projection: Perspective
+  )
+  camera.setCameraMode(FIRST_PERSON)
+
+  let imMap:Image = loadImage("resources/cubicmap.png") # load cubicmap image (RAM)
+  let cubicMapTexture:Texture2D = loadTextureFromImage(imMap)  # convert image to texture (VRAM)
+  let mesh = genMeshCubicmap(imMap, ( x:1.0, y:1.0, z:1.0))
```

Comment:
Let's use consistent style. Space between `:` and everything else, space between `(` and everything else. This is a good example of what style to use:
```nim
let playerPos:Vector2 = Vector2(x: camera.position.x, y: camera.position.z)
```

There are a lot of inconsistencies

Reply:
ok for space after tuple/object name, but are your sure for a space between `(` and the rest ? I see i have forgotten to remove the one here as you had asked me  last day (but it was for function calling, so may be different here ?)

Reply:
I see you used nimpretty, it generally should solve the problem of how to style it correctly. But sometimes it does strange indentation that should be fixed manually. In this case I think it's good enough